### ISSUE TITLE
Plugin request for Buf Schema Registry - `protoc-gen-bq-schema`

### ISSUE THREAD

**Issue by krak3n** [bufbuild/plugins#317]
## Mandatory

**Where is the source code for the plugin?**

The source code for the `protoc-gen-bq-schema` plugin is found here:

https://github.com/GoogleCloudPlatform/protoc-gen-bq-schema

## Optional

**Does the plugin have a valid semver version?**

Yes, there is a `1.0.0`: https://github.com/GoogleCloudPlatform/protoc-gen-bq-schema/releases/tag/v1.0

**Does this plugin have runtime dependencies?**

Example, the generated code for the `protoc-gen-bq-schema` plugin has a runtime dependency on Go 1.15 the modules defined in it's `go.mod`:

```
	github.com/golang/glog v1.0.0
	github.com/golang/protobuf v1.5.2
	google.golang.org/protobuf v1.28.0
```

**Does the plugin have a dependency on another plugin?**

I don't believe so.

**Do you think this plugin will be compatible with the BSR go module proxy or npm registry?**

I think so.

**Comment by mfridman** [comment]
Hey @krak3n, thanks for the suggestion.

We've opted to publish a more refined version of this plugin (among others, such as JSON Schema and PubSub). The goal with these new plugins is to address a few bugs and open issues, as well as document [limitations](https://github.com/bufbuild/protoschema-plugins?tab=readme-ov-file#limitations).

I'm going to close this issue in favor of https://github.com/bufbuild/plugins/issues/1113. Happy to address any questions or concerns.

**Comment by krak3n** [comment]
Ok thanks @mfridman, looks like a good solution 👍

**Comment by mfridman** [comment]
Note, we ended up publishing this plugin after all, available on the BSR as a remote plugin:

https://buf.build/googlecloudplatform/bq-schema

For an explanation of why, see https://github.com/bufbuild/protoschema-plugins/issues/6#issuecomment-2040358836

TL;DR - there wasn't a big enough differentiator at the moment. Hope this isn't any trouble and if there's any issues we're more than happy to help.
